src/box/index.h

```diff
@@ -243,6 +243,7 @@ box_index_count(uint32_t space_id, uint32_t index_id, int type,
  * @param tuple Tuple from which need to extract key.
  * @param space_id Space identifier.
  * @param index_id Index identifier.
+ * @param key_size
```

Comment:
Same here, I think it is better to actually document parameters.

It is how it is documented in the `tuple_extract_key()` definition.

```c
 * @param key_size - here will be size of extracted key
```

First, I would mark the parameter as the output parameter: `@param[out]`. Next, I would mention that NULL can be passed here if the key size is not needed.

For example:

```c
 * @param[out] key_size where to store the size of the extracted
 *                      key; pass NULL here if it is not needed
```

Reply:
Updated, thanks:

```diff
--- a/src/box/index.h
+++ b/src/box/index.h
@@ -243,7 +243,8 @@ box_index_count(uint32_t space_id, uint32_t index_id, int type,
  * @param tuple Tuple from which need to extract key.
  * @param space_id Space identifier.
  * @param index_id Index identifier.
- * @param key_size
+ * @param[out] key_size where to store the size of the extracted
+ *             key; pass NULL here if it is not needed
  * @retval not NULL Success
  * @retval     NULL Memory Allocation error
  */
```